catalog.html

```diff
@@ -24,154 +23,172 @@ <h1>
      </h1>
     </div>
     <div>
-     +7 (812) 555-05-55<br>
-     г.Санкт-Петербург, ул. Б. Конюшенная, д. 19/8
+      <a href="tel:+78125550555">+7 (812) 555-05-55</a><br>
+      <address>г.Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</address>
     </div><br>
+    <a href="#"><img src="img/icon-user.svg" alt="Икнока пользователя для входа"></a>
     <button>Равшан Джамшутович</button>
+    <a href="#"><img src="img/icon-login.svg" alt="Иконка выхода из личного кабинета"></a>
```

Comment:
```suggestion
    <a href="#">Равшан Джамшутович</a>
    <a href="#"><span class="visually-hidden">Выход из личного кабинета</span></a>
```
Это 2 ссылки, кнопка здесь не нужна. Иконки являются декоративными элементами.